libjolie/src/main/java/jolie/lang/CodeCheckMessage.java

```diff
@@ -77,7 +77,13 @@ public String toString() {
 			} else {
 				messageBuilder.append( "No descriptive error message found.\n" );
 			}
-			messageBuilder.append( myContext.line() ).append( ':' ).append( myContext.lineString() ).append( '\n' );
+			if( myContext.lineString().contains( myContext.line() + ":" ) ) {
+				messageBuilder.append( myContext.lineString() ).append( '\n' );
+			} else if( (myContext.lineString().split( "\n" )).length > 1 ) {
```

Comment:
Update `ParsingContext` with a method that gives you the list of strings and make a static method in `CodeCheckMessage` that puts the line numbers in.